src/app-layout/visual-refresh/context.tsx

```diff
@@ -435,9 +435,7 @@ export const AppLayoutInternalsProvider = React.forwardRef(
     const drawersTriggerCount =
       drawers.length + (splitPanelDisplayed && splitPanelPosition === 'side' ? 1 : 0) + (!toolsHide ? 1 : 0);
     const hasOpenDrawer =
-      activeDrawerId !== undefined ||
-      isToolsOpen ||
```

Comment:
We should remove `isToolsOpen` from all such conditions, because "tools" -> "drawer" conversion happens once, inside the `useDrawers` hook. All other code should treat tools as a normal drawer.